com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -70,6 +70,25 @@ public void CopyActions(ActionBuffers actionBuffers)
         }
     }
 
+    /// <summary>
+    /// Simple wrapper around VectorActuator that overrides GetBuiltInActuatorType
+    /// so that it can be distinguished from a standard VectorActuator.
+    /// </summary>
+    internal class AgentVectorActuator : VectorActuator
```

Comment:
This can be undone, but I wanted to be able to distinguish the actuator that all agents have from a VectorActuator that might have been created some other way.

Reply:
seems like a good idea to me